utils.h

```diff
@@ -176,7 +175,12 @@ static inline int is_cue_url(const char *name)
 
 static inline int is_url(const char *name)
 {
-	return is_http_url(name) || is_cdda_url(name) || is_cue_url(name);
+	return is_http_url(name) || is_cdda_url(name);
+}
+
+static inline int is_partial_url(const char *name)
```

Comment:
Yes, I've been thinking about usefulness of these functions.
They have no real purpose: It's just an alias to checkers. At first, it was needed to avoid "cue" and "cdda" mentions in the main code. Readability, you know...

But this code (is_url, is_partial_url) isn't nice for me. Maybe there is another way to get rid of those bad "link" hacks?